### PR TITLE
Suspend side effects in Eff

### PR DESCRIPTION
**Pull request by vasilmkd**
A bit of bikeshedding and pretending to do work.

cc @armanbilge

### CODE REVIEW

File: webworker-tests/src/main/scala/cats/effect/IOSpecRunner.scala [hunk 1]
```diff
@@ -42,9 +42,8 @@ object IOSpecRunner extends IOApp.Simple with ClassRunner {
       val action = for {
         printers <- createPrinters(env.arguments, loader).toAction
         stats <- Runner.runSpecStructure(spec.structure(env), env, loader, printers)
-        // TODO I have no idea how to suspend effects in this
-        _ = postMessage(stats.toString)
-        _ = postMessage(stats.isSuccess)
+        _ <- Actions.delayed(postMessage(stats.toString))
+        _ <- Actions.delayed(postMessage(stats.isSuccess))
```
Comment (armanbilge) [inline webworker-tests/src/main/scala/cats/effect/IOSpecRunner.scala]: lol that wasn't too bad! Thanks :D

Comment (armanbilge) [reply webworker-tests/src/main/scala/cats/effect/IOSpecRunner.scala]: Actually while you're here you can delete line 45, the stats are printed out during the test run actually.

Comment (armanbilge) [reply webworker-tests/src/main/scala/cats/effect/IOSpecRunner.scala]: This is the output of that line: https://github.com/typelevel/cats-effect/pull/2219/checks?check_run_id=3306036979#step:9:2206

Comment (vasilmkd) [reply webworker-tests/src/main/scala/cats/effect/IOSpecRunner.scala]: Make a suggestion by changing it to an empty line, I will merge it.

Comment (armanbilge) [reply webworker-tests/src/main/scala/cats/effect/IOSpecRunner.scala]:
```suggestion
        _ <- Actions.delayed(postMessage(stats.isSuccess))
```

**Review by armanbilge — APPROVED** [review]

**Review by djspiewak — APPROVED** [review]

**Pull request merged**